webapp/packages/plugin-data-viewer/src/DatabaseDataModel/DataContext/DATA_CONTEXT_DV_RESULT_KEY.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { createDataContext } from '@cloudbeaver/core-data-context';
+
+import type { IResultSetElementKey } from '../Actions/ResultSet/IResultSetDataKey.js';
+
+export const DATA_CONTEXT_DV_RESULT_KEY = createDataContext<IResultSetElementKey>('data-viewer-database-result-key');
```

Comment:
should be somewhere in the result set related folder (it's not global context, it's result set specific)